ui/lib/core/addon/templates/components/replication-mode-summary.hbs

```diff
@@ -101,7 +101,7 @@
   </div>
   <div class="level-right">
     {{#if replicationDisabled}}
-      {{#link-to "vault.cluster.replication.mode.index" cluster.name mode class="button is-primary"}}
```

Comment:
because we are using peekRecord, this was causing a console error.  I can't exactly explain why the peekRecord now adds a vault.cluster.replication to the link to helper here, but it does.  Here was the error, you can see the link-to is trying to hit vault.cluster.replication.vault.cluster.replication.mode.index
![image](https://user-images.githubusercontent.com/6618863/82490316-52092f80-9aa0-11ea-8d10-750035741b52.png)

Reply:
👏